.gitignore

```diff
@@ -1,5 +1,5 @@
 .deps
-contracts/SuperChamps/artifacts/*
-contracts/synthetix/contracts/artifacts/*
-contracts/synthetix/contracts/interfaces/artifacts/*
-*.json
```

Comment:
*.json should be git ignored?

Reply:
This could be made less broad if needed. But I don't want all of the build artifacts to be included in the repo.